iosapp/app/Views/RunningView.swift

```diff
@@ -71,7 +70,7 @@ struct RunningView: View {
           for: .seconds(0.5)
         )
 
-        Text("\(self.device.vendorId?.uuidString.lowercased() ?? "unknown")")
+        Text("\(UIDevice.current.identifierForVendor?.uuidString.lowercased() ?? "unknown")")
```

Comment:
because the way to get the vendor id is restricted to the main actor, i had to make the dependency vend an async function, which is awkward to use here in the view. so, since the view is by definition MainActor, i just reach out to the underlying value directly here.